backend/index.js

```diff
@@ -9,15 +9,37 @@ const io = require("socket.io")(server, {
         methods: ["GET", "POST"]
     }
 });
-const port = 4001;
+const MafiaGame = require("./domain/MafiaGame");
+const Player = require("./domain/Player");
+
 
 
 // Listen for a "connection" event for incoming sockets.
 io.on("connection", (socket) => {
     console.log("User has connected");
+/*********************  Create Lobby and lobby code *******/
```

Comment:
Would we be able to do something like this? https://stackoverflow.com/questions/24153138/socket-io-best-coding-practice/52595646. Otherwise `Index.js` would get pretty big with all the other event handlers.